readme.md

```diff
@@ -58,3 +64,7 @@ client.createEmbeddings(params: EmbeddingParams): Promise<EmbeddingResponse>
 ### Type Definitions
 
 The type definitions are avaible through TSServer, and can be found here: [type definitions](/src/types.ts).
+
+## License
+
+MIT © [Dexa](https://dexa.ai)
```

Comment:
This should probably match the license file right? We can change both to Dexa.

Reply:
sounds good; yeah, I had thought about changing the license file, but I hesitated cause I didn't want you to think I was removing your name from the project :)

Reply:
haha sounds good